llvm/test/CodeGen/NVPTX/reduction-intrinsics.ll

```diff
@@ -0,0 +1,1908 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --extra_scrub --version 5
```

Comment:
Please break this PR into 2 commits, one which pre-commits the tests and a second which contains your changes and updates your tests.